.github/workflows/bot_fix_style_comment.yaml

```
@@ -70,14 +68,12 @@ jobs:
 
       - name: lint references.bib
         if: steps.user_permission.outputs.require-result == 'true'
-        continue-on-error: true  # allows the following `reviewdog` step to add GitHub suggestions
         run: |
-          ./scripts/lint-bib.sh
+          ./scripts/lint-bib.sh || true
```

Let's retain the comment:
```suggestion
          # ignoring the return code allows the following `reviewdog` step to add GitHub suggestions
          ./scripts/lint-bib.sh || true
```